wemake_python_styleguide/visitors/tokenize/syntax.py

```diff
@@ -49,3 +60,11 @@ def _check_line_starts_with_dot(self, token: tokenize.TokenInfo) -> None:
         line = token.line.lstrip()
         if line.startswith('.') and not line.startswith('...'):
             self.add_violation(LineStartsWithDotViolation(token))
+
+    def _check_line_comprise_carriage_return(
+        self, token: tokenize.TokenInfo,
```

Comment:
My wild guess that it should be `token.string.startswith('\r')`
Because here's how our token looks like: `TokenInfo(type=4 (NEWLINE), string='\r\n', start=(1, 5), end=(1, 7), line='1 + 2\r\n')`

Reply:
Next test was failed: string.startswith() also unuseful in this case. Or not? Can you see the code fixedly, eh?

Reply:
Now (Mon, Feb 24) My local pytest under poetry's venv give me adecvate results with 3 Fails on 3 negative testscases and 2 Passed on positive cases.